elementor-localization.php

```diff
@@ -25,7 +27,11 @@
  */
 if ((defined('DOING_CRON') && true === DOING_CRON) || is_admin()) {
     add_action('smartling_before_init', static function ($di) {
-        Bootloader::boot(__FILE__, $di);
+        try {
+            Bootloader::boot(__FILE__, $di);
+        } catch (\Error $e) {
```

Comment:
It would be nice to dump the error `$e` somewhere

Reply:
Done, best effort to write to logs